setup.py

```diff
@@ -231,7 +231,6 @@ def run(self):
     'python-lsp-black>=2.0.0,<3.0.0',
     'pyls-spyder>=0.4.0',
     'pyqt5>=5.10,<5.16',
-    'pyqtwebengine>=5.10,<5.16',
```

Comment:
i think this needs to be added back somewhere, as an optional dependency or something? not sure if you have a `spyder[full]` or the like.

Reply:
or at least it should be addressed so that users from pypi still get "help". 

again, i don't want to "remove features from spyder to accomodate conda-forge".

Reply:
Do we really need to remove this? Since we can declare the dependencies we want in conda-forge, why does it matter to have a different set of dependencies in PyPI?